test_scenarios/T4_Activate_CT.md

```diff
@@ -54,7 +55,39 @@ ___WARUNEK SUKCESU:___
 5. __App User__ wybiera opcję "OK"
 6. System wyświetla informację o poprawnym uruchomieniu CT "Test Task 01".
 7. __App User__ wybiera opcję "OK"
+8. __System__ wyświetla widok CT "Test Task 01" ze statusem "IN_PROGRESS"
+9. __App User__ wybiera opcję "Odśwież logi" po około 10 sekundach
+10. __System__ wyświetla log z aktualną godziną i treścią "OK"
+11. System wyświetla status "DONE"
+
+___WARUNEK SUKCESU:___
+
+- Do domyślnego loggera zostało wypisane "OK".
```

Comment:
Jeszcze można dodać tak jak w następnym, że status CT się zmienił w warunkach sukcesu

Reply:
Tak. Trzeba jeszcze dać sprawdzanie zmiany statusu maszyny w MM UI, bo o to Chabrowi też chodziło przede wszystkim.